Makefile

```diff
@@ -21,7 +21,7 @@ TAG_LATEST ?= false
 LOCALIP ?= $(shell ifconfig | grep inet | grep -v '::' | grep -v 127.0.0.1 | head -n1 | awk '{print $$2}')
 
 # Sets GIT_REF to a tag if it's present, otherwise the short rev.
-GIT_REF = $(shell git describe --tags || git rev-parse --short=8 --verify HEAD)
+GIT_REF = $(shell git describe --tags --exact-match 2>/dev/null || git rev-parse --short=8 --verify HEAD)
```

Comment:
As per the main discussion, lets update this to:
```
git describe --tags --exact-match 2>/dev/null || basename $(git describe --all --long)
```

Please update the comment too.